pages/docs/integrations/grouparoo-intercom.mdx

```diff
@@ -11,132 +11,195 @@ Grouparoo's [Intercom](https://www.intercom.com/) Plugin enables you to export c
 
 This guide will show you how to work with the Intercom Plugin to [create a Destination](#create-an-intercom-destination) to export your data.
 
-<PluginDocsInstallation plugin="Intercom" />
+## Overview
 
-<PluginDocsCreateApp plugin="Intercom" />
+Intercom is a messaging platform for customer communications. You can use Intercom to stay in touch with customers. Grouparoo can help you to sync data from your warehouse to Intercom easily.
 
-<ValidateAndApplyConfig headingLevel="h3" />
+### Destinations
 
-## Create an Intercom Destination
+<table className="docTable">
```

Comment:
Nice looking tables!

Reply:
Submitted for re-review with the feedback and an added description!